Documentation/ApiOverview/CommandControllers/Tutorial.rst

```diff
@@ -0,0 +1,225 @@
+.. include:: /Includes.rst.txt
+
+.. _console-command-tutorial:
+
+========
+Tutorial
+========
+
+Create a console command from scratch
+=====================================
+
+A console command is always situated in an extension. If you want to create
+one, :ref:`kickstart a custom extension <extension-kickstart>` or use your
+sitepackage extension.
+
+Creating a basic command
+========================
+
+The extension kickstarter ":ref:`Make <extension-make>`" offers a convenient
+console command that creates a new command in an extension of your choice:
+:ref:`Create a new console command with "Make" <extension-make-console-command>`.
+You can use "Make" to create a console command even if your extension was
+created by different means.
+
+This command can be found in the
+`Examples extension <https://github.com/TYPO3-Documentation/t3docs-examples>`__.
+
+1. Register the command
+-----------------------
+
+Register the command in :file:`Configuration/Services.yaml` by adding the service
+definition for your class as tag :yaml:`console.command`:
+
+..  code-block:: yaml
+    :caption: EXT:examples/Configuration/Services.yaml
+    :linenos:
+    :emphasize-lines: 11-16
+
+    services:
+      _defaults:
+        autowire: true
+        autoconfigure: true
+        public: false
+
+      T3docs\Examples\:
+        resource: '../Classes/*'
+        exclude: '../Classes/Domain/Model/*'
+
+      T3docs\Examples\Command\DoSomethingCommand:
+        tags:
+          - name: console.command
+            command: 'examples:dosomething'
+            description: 'A command that does nothing and always succeeds.'
+
+..  note::
+    Despite using :file:`autoconfigure: true` the commands
+    have to be explicitly defined in :file:`Configuration/Services.yaml`. The
+    description is mandatory.
```

Comment:
```suggestion
    have to be explicitly defined in :file:`Configuration/Services.yaml`. It
    is recommended to always supply a description, Otherwise there is 
    an empty space in the list of commands.
```